data/gaomon-pd1560.tablet

```diff
@@ -0,0 +1,24 @@
+# GAOMON
+# PD1560
+#
+# This file is a slighlty modified gaomon-s620.tablet
+
+[Device]
+Name=GAOMON PD1560
+ModelName=
+DeviceMatch=usb:256c:006e
```

Comment:
you'll need the name here as well to match correctly, unfortunately that USB ID is re-used my multiple devices, without the name match we cannot be correct here. See the S620 file for details.